src/Utilities/Blaze.hpp

```diff
@@ -18,15 +21,9 @@
 
 // Override padding, streaming and kernel options
 #define _BLAZE_SYSTEM_OPTIMIZATIONS_H_
-namespace blaze {
-constexpr bool usePadding = false;
-constexpr bool useStreaming = true;
-constexpr bool useOptimizedKernels = true;
-}
 
-namespace blaze {
-constexpr bool useDefaultInitialization = false;
-}
+const blaze::AlignmentFlag blaze_unaligned = blaze::AlignmentFlag::unaligned;
+const blaze::PaddingFlag blaze_unpadded = blaze::PaddingFlag::unpadded;
```

Comment:
Not sure these two variables are necessary. I find `blaze::AlignmentFlag::unaligned` as clear as `blaze_unaligned` and not too much longer, and you avoid the level of indirection.